en/202407/exploring-geo-distribution-in-emqx-for-enhanced-scalability.md

```diff
@@ -0,0 +1,258 @@
+
+It's important to mention that, as always, there are apparent trade-offs of this deployment model: the cluster needs more computing resources and becomes less resilient to failures due to all the core nodes residing in the same data center. Nevertheless, given the benefits, it is a trade-off worth making. Furthermore, thanks to new configuration options and the new topic index design, we could reasonably expect poor subscription latency will be offset by (theoretically, almost unlimited) throughput gains. Consider trying out recent [EMQX 5.7.0](https://github.com/emqx/emqx/releases/tag/v5.7.0) release that has all these features configurable and well-tested.
+
+If you find yourself deploying this kind of cluster for your geo-distributed application, definitely consider further [tuning the configuration](https://docs.emqx.com/en/emqx/v5.7/configuration/configuration.html). There are now a couple of extra configuration parameters worth tweaking to improve the performance of a multi-region EMQX cluster. Here is an `emqx.conf` example with parameters applicable to both core and replicant nodes combined for simplicity.
+
+```shell
+# Core nodes
+node.default_bootstrap_batch_size = 10000
+# Replicants
+node.channel_cleanup_batch_size = 10
+```
+
+Now for the brief overview of what those parameters affect.
+
+- `node.channel_cleanup_batch_size`
+
+  Relevant for replicant nodes. If the network latency is high, significantly reducing this value from the default 10,000 improves performance during an abrupt disconnect of large numbers of clients.
+
+- `node.default_bootstrap_batch_size`
+
+  Relevant for core nodes. The default value is 500, but it can be greatly increased to reduce the time it takes for a replicant node to join a cluster with many active subscriptions.
+
+## Alternatives
+
+Nevertheless, there are situations where latency this high is not acceptable. EMQX has a solution for these situations: rather than setting up a geo-distributed cluster, it is possible to deploy a separate independent cluster per each region and connect those clusters through [asynchronous MQTT bridges](https://docs.emqx.com/en/emqx/v5.7/data-integration/data-bridge-mqtt.html). It is a radically different deployment model, which demands additional compute resources and operational overhead. However, it offers a clear advantage: the shared state does not need to be consistently maintained across three continents. Each cluster will have its own state, and any changes induced by subscriptions will be handled locally, making the latency low.
```

Comment:
```suggestion
Nevertheless, there are situations where latency this high is not acceptable. EMQX has a solution for these situations: rather than setting up a geo-distributed cluster, it is possible to deploy a separate independent cluster per region and connect those clusters through [asynchronous MQTT bridges](https://docs.emqx.com/en/emqx/v5.7/data-integration/data-bridge-mqtt.html). It is a radically different deployment model, which demands additional compute resources and operational overhead. However, it offers a clear advantage: the shared state does not need to be consistently maintained across three continents. Each cluster will have its own state, and any changes induced by subscriptions will be handled locally, making the latency low.
```